src/index.mjs

```diff
@@ -14,7 +14,19 @@
 
 const handleRequest = async (request, env, ctx) => {
   const url = new URL(request.url);
-
+  if (url.port) {
```

Comment:
this would generate a redirect if the url were e.g. `https://www.acme.org:443/`. intentional?